research/src/components/breadcrumb-anatomy.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+
+import React from 'react'
+import './anatomy.css'
+import './breadcrumb-anatomy.css'
+
+const BreadcrumbAnatomy = () => {
+  return (
+    <div class="component-anatomy-wrapper">
+      <input type="checkbox" id="show-slots" />
+      <label for="show-slots"> Show slots</label>
+      <div class="component-anatomy">
+        <host name="breadcrumb" role="nav">
+          <slot name="breadcrumb-container">
+            <part name="ol">
+              <part name="li">item</part>
```

Comment:
as noted in the other comment add into the name `[part=breadcrumb-item]` to both the `ol` and `li`